surfsense_backend/app/tasks/connectors_indexing_tasks.py

```
@@ -912,3 +915,192 @@ async def index_linear_issues(
+        )
+        connector = result.scalars().first()
+        
+        if not connector:
+            return 0, f"Connector with ID {connector_id} not found or is not a Discord connector"
+        
+        # Get the Discord token from the connector config
+        discord_token = connector.config.get("DISCORD_BOT_TOKEN")
+        if not discord_token:
+            return 0, "Discord token not found in connector config"
+        
+        logger.info(f"Starting Discord indexing for connector {connector_id}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)